build_tools/requirements_conda.txt

```diff
@@ -7,3 +7,4 @@ nltk==3.2.1
 pytest==2.9.1       
 flask==0.12         # server
 requests==2.11.1
+matplotlib==1.5.3
```

Comment:
@DenisFLASH  Essentually we don't want matplotlib in the core dependencies (and there is no `requirements_pip.txt` anymore just a `requirements.txt` (in the current version of the code).. . Maybe we can just manually add `pip install matplotlib==1.5.3` to `.travis.yml`  and `appveyor.yml` (after the tests are run, but before the examples are executed)?